apps/authentication/models/connection_token.py

```diff
@@ -306,3 +307,14 @@ class AdminConnectionToken(ConnectionToken):
     class Meta:
         proxy = True
         verbose_name = _("Admin connection token")
+
+    @lazyproperty
+    def actions(self):
+        return ActionChoices.all()
+
+    @lazyproperty
+    def expire_at(self):
+        return (timezone.now() + timezone.timedelta(days=365)).timestamp()
+
+    def is_valid(self):
+        return True
```

Comment:
The provided code does not contain any specific technical difficulties that could cause an issue. It seems to be functioning as expected based on its current implementation.

There were no syntax errors found, and there didn't seem to be any obvious logical errors present in the provided code at this time. Therefore, it's safe to assume it will run without problems under normal circumstances. If you have more questions about the code or need additional information, feel free to ask!